### PR TITLE
108 fc7 create task modal content

### PR DESCRIPTION
**Pull request by AleBarboni**

### CODE REVIEW

**Review by soringifei — APPROVED** [review]

File: src/components/CreateTaskItems.jsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const OPTIONS = [
```
Comment (cezarmocanu) [inline src/components/CreateTaskItems.jsx:1]: This approach of creating constants is not correct:
- jsx files are expected to have a jsx return function iside them such as a react component, if there is no jsx the file should be js
- the file is in the components folder even though it is not a component
- the name OPTIONS is too general an can be easily mixed up

Usually constants should be added in the constants folder, as a .js file. In this particular case, there is no need for other components to use the OPTIONS, so the best approach is to leave this variable in the component file(modal-content/CreateTask) right above the component function

File: src/components/modal-content/CreateTask.jsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+	Grid,
+} from "@mui/material";
+import CloseIcon from "@mui/icons-material/Close";
+
+import Button from "./../shared/button/Button";
+import FormLabel from "./../shared/input/FormLabel";
+import TextField from "./../shared/input/TextField";
+import DatePicker from "./../shared/date-picker/DatePicker";
+
+import OPTIONS from "../CreateTaskItems";
```
Comment (cezarmocanu) [inline src/components/modal-content/CreateTask.jsx:22]: Please remove the import and add the options as a local constant

**Review by cezarmocanu — CHANGES_REQUESTED** [review]

File: src/components/modal-content/CreateTask.jsx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+						<Stack direction={"row"}>
+							<Grid container spacing={2}>
+								<Grid item xs={6}>
+									<Stack gap={1}>
+										<FormLabel>
+											<Typography fontWeight="medium">Task Priority</Typography>
+										</FormLabel>
+										<TextField select>
+											{OPTIONS.map((option) => (
+												<MenuItem key={option.label} value={option.label}>
```
Comment (cezarmocanu) [inline src/components/modal-content/CreateTask.jsx:69]: Please also check this issue in the console
image

**Review by cezarmocanu — CHANGES_REQUESTED** [review]

**Review by HalaszPaul — APPROVED** [review]

**Review by soringifei — APPROVED** [review]

**Review by ovidiufilip10 — APPROVED** [review]

File: src/components/modal-content/CreateTask.jsx (hunk 3 above)
Comment (AleBarboni) [reply src/components/modal-content/CreateTask.jsx:71]: done

**Review by cezarmocanu — APPROVED** [review]

**Pull request merged**